src/excmds.ts

```diff
@@ -2900,20 +2900,14 @@ export async function hint(option?: string, selectors?: string, ...rest: string[
         let link = await hinting.pipe_elements(DOM.anchors())
         anchorUrl.hash = link.id || link.name
         run_exstr("yank " + anchorUrl.href)
-    } else if (option === "-c") DOM.simulateClick(await hinting.pipe(selectors))
+    } else if (option === "-c") DOM.simulateClick(await hinting.pipe(selectors)[0])
     // Deprecated: hint exstr
-    else if (option === "-W") run_exstr(selectors + " " + rest.join(" ") + " " + (await hinting.pipe(DOM.HINTTAGS_selectors)))
-    else if (option === "-pipe") return (await hinting.pipe(selectors))[rest.join(" ")]
+    else if (option === "-W") run_exstr(selectors + " " + rest.join(" ") + " " + (await hinting.pipe(DOM.HINTTAGS_selectors)[0]))
+    else if (option === "-pipe") return (await hinting.pipe(selectors)[0])[rest.join(" ")]
```

Comment:
[0] needs to go on the other side of the bracket - so (await ...)[0]. I think you've made that mistake in quite a few places.

Reply:
Yeah, sorry about that. I should have tested this more.